### PR TITLE
Issue #163: Included documentation for to_array method.

### PR DESCRIPTION
**Pull request by krisstee**
[ ] Wrote test for feature
[ ] Added changes to CHANGELOG.md
[ ] Bumped version number (delete if unneeded)

**Changes proposed:**
<description of the PR>

Changes include documentation added for the to_array method. Closed previous PR to stay consistent with the data-8/datascience branch and remove changes to whitespaces.
This PR should only include additions to the to_array method.

### CODE REVIEW

**Comment by krisstee** [comment]
I originally specified dimensions in what was returned in the method, but removed it as it's dependent on the table

File: datascience/tables.py [hunk 1]
```diff
@@ -2655,7 +2655,42 @@ def to_csv(self, filename):
         self.to_df().to_csv(filename, index=False)
 
     def to_array(self):
-        """Convert the table to a structured NumPy array."""
+        """Convert the table to a structured NumPy array.
+
+        The elements in the resulting array preserve row and column replacement
```
Comment (davidwagner) [inline datascience/tables.py:2660]: I did not  understand what "row and column replacement" means.  Actually, I'm having a hard time understanding this entire paragraph.  Can we reword?  Maybe the result is an array of rows, where each row is represented as a sequence of fields, and each field corresponds to a column label?  Or something?

File: datascience/tables.py [hunk 2]
```diff
@@ -2655,7 +2655,42 @@ def to_csv(self, filename):
...
+        in the table. The array's dtype attribute reflects both column names and
+        datatype.
+
+        Args:
+            None
+
+        Returns:
+            arr: a NumPy array
+
+        The following is an example of calling to_array()
```
Comment (davidwagner) [inline datascience/tables.py:2670]: Good example!

File: datascience/tables.py (hunk 1 above)
Comment (krisstee) [reply datascience/tables.py]: I meant to just type "row and column placement", but I just pushed a commit which the description is much simpler.

**Review by adnanhemani — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**